package.json

```diff
@@ -34,6 +34,7 @@
     "inquirer": "2.0.0"
   },
   "devDependencies": {
+    "eslint-plugin-html": "3",
```

Comment:
What is this doing, and why is it needed?

Reply:
I have no idea. At first I encountered this error when I run test
```
> gulpfile-generator@1.3.3 test /home/zerox/Desktop/projects/gulpfile-generator
> NODE_ENV=test gulp

[08:17:15] Using gulpfile ~/Desktop/projects/gulpfile-generator/gulpfile.js
[08:17:16] Starting 'set-test-env'...
[08:17:16] Finished 'set-test-env' after 85 μs
[08:17:16] Starting 'json'...
[08:17:16] Finished 'json' after 8.48 ms
[08:17:16] Starting 'lint'...
[08:17:16] Finished 'lint' after 28 ms
[08:17:16] Starting 'test'...

events.js:160
      throw er; // Unhandled 'error' event
      ^
Error: Failed to load plugin html: Cannot find module 'eslint-plugin-html'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.resolve (internal/module.js:27:19)
    at Plugins.load (/home/zerox/Desktop/projects/gulpfile-generator/node_modules/eslint/lib/config/plugins.js:106:29)
    at Array.forEach (native)
    at Plugins.loadAll (/home/zerox/Desktop/projects/gulpfile-generator/node_modules/eslint/lib/config/plugins.js:146:21)
    at loadFromDisk (/home/zerox/Desktop/projects/gulpfile-generator/node_modules/eslint/lib/config/config-file.js:492:35)
    at Object.load (/home/zerox/Desktop/projects/gulpfile-generator/node_modules/eslint/lib/config/config-file.js:550:20)
    at Config.getLocalConfigHierarchy (/home/zerox/Desktop/projects/gulpfile-generator/node_modules/eslint/lib/config.js:228:44)
    at Config.getConfigHierarchy (/home/zerox/Desktop/projects/gulpfile-generator/node_modules/eslint/lib/config.js:180:43)
    at Config.getConfigVector (/home/zerox/Desktop/projects/gulpfile-generator/node_modules/eslint/lib/config.js:287:21)
npm ERR! Test failed.  See above for more details.

```
I resolved it by installing `eslint-plugin-html` using
```
yarn add eslint-plugin-html -D 
```

That's why it's there

Reply:
Strange, I can run tests without it being present. But if you can't let's leave it there.